src/Entity/ProcurationProxy.php

```diff
@@ -585,17 +611,25 @@ public function getAvailableRoundsAsString(): string
             return self::NO_AVAILABLE_ROUND;
         }
 
-        return implode("\n", $availableRounds->toArray());
+        return implode('\n', $availableRounds->toArray());
+    }
+
+    public function getFoundRequests(): Collection
+    {
+        return $this->foundRequests;
     }
 
-    public function getFoundRequest(): ?ProcurationRequest
+    public function addFoundRequest(ProcurationRequest $procurationRequest): void
```

Comment:
why do we need adder and remover methods? 
we control this with `process` and `unprocess` methods no?

Reply:
the `process()` and `unprocess()` function will use those functions (`addFoundRequest()` & `removeFoundRequest`) right after checking if it's possible or not

Reply:
so they can be private to avoid using it?